apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/index.tsx

```diff
@@ -6,13 +6,26 @@ import { Browse } from "./Browse";
 import { useTranslation } from "react-i18next";
 import { useCatalog, useRecentlyUsedDB } from "../hooks";
 import { LocalLiveAppSection } from "./LocalLiveAppSection";
+import { useLocation } from "react-router-dom";
+import { Categories } from "@ledgerhq/live-common/wallet-api/react";
 
 export function Catalog() {
+  const { t } = useTranslation();
   const recentlyUsedDB = useRecentlyUsedDB();
+  const { state } = useLocation() as ReturnType<typeof useLocation> & {
+    state?: {
+      category?: Categories["selected"];
+    };
+  };
+  const deeplinkInitialCategory = state?.category;
```

Comment:
Location state with our usage of the hash router can be unreliable, as it might be reset on navigation
You should already have the information you want in the search field instead

Reply:
Nice one, I'll just use the `search` instead